### DISCUSSION TITLE
Allowlist format

### DISCUSSION THREAD

**Discussion by timkgh** [AdguardTeam/AdGuardHome#3700] [Q&A] [answered]
"DNS allowlists" settings page says "AdGuard Home understands basic adblock rules and hosts files syntax."

But the hosts file syntax doesn't make a lot of sense for allow lists because there are no IPs to list. Unless AGH just ignores the IPs and uses the exact domains only.

Will this list work directly or does it need to be processed to change the format to basic ABP?
https://raw.githubusercontent.com/nextdns/metadata/master/privacy/affiliate-tracking-domains

It appears to work, but not sure about all entries, in particular those with comments at the end of the line.

Can we please have better documentation with some example lists?

**Comment by timkgh** [comment]
Maybe the help text for allowlists is supposed to say "domains-only syntax" instead of hosts files?
https://github.com/AdguardTeam/AdGuardHome/wiki/Hosts-Blocklists#domains-only

**Comment by timkgh** [comment]
@ameshkov is this just a documentation issue?

**Comment by ameshkov** [comment, marked as answer]
> But the hosts file syntax doesn't make a lot of sense for allow lists because there are no IPs to list. Unless AGH just ignores the IPs and uses the exact domains only.

That's exactly what AGH does.

> It appears to work, but not sure about all entries, in particular those with comments at the end of the line.

These will work just okay.

> Maybe the help text for allowlists is supposed to say "domains-only syntax" instead of hosts files?

It supports both.

Also, I've updated examples on Wiki to reflect what's considered comments.

**Reply by timkgh** [reply]
Thank you.

**Comment by timkgh** [comment]
@ameshkov having a problem with the list referenced above, see https://github.com/AdguardTeam/AdGuardHome/issues/3741
